app/views/liquids/_stackexchange.html.erb

```diff
@@ -32,8 +32,6 @@
     </p>
   </div>
   <div class="ltag__stackexchange--btn--container">
-    <button class="ltag__stackexchange--btn" type="button">
-      <a href="<%= post_url %>" target="_blank" rel="noopener noreferrer"><%= t("views.liquids.stackexchange.open.#{post_type}") %></a>
-    </button>
+    <a href="<%= post_url %>" class="ltag__stackexchange--btn" target="_blank" rel="noopener noreferrer"><%= t("views.liquids.stackexchange.open.#{post_type}") %></a>
```

Comment:
I love that this is no longer a button 😄  It still _looks_ like a button, which isn't ideal, but it's definitely a step forward ✨